widgets/DcaWizard.php

```diff
@@ -148,8 +148,10 @@ public function generate()
         $blnShowOperations = $this->showOperations;
         $widget = $this;
 
+        // check for custom template
+        $strTemplate = ($this->customTpl) ?: 'be_widget_dcawizard';
 
-        $objTemplate = new BackendTemplate('be_widget_dcawizard');
+        $objTemplate = new BackendTemplate($strTemplate);
```

Comment:
Can just use `$this->customTpl ?: 'be_widget_dcawizard'` right away here, no need for a local variable :)